app/main.py

```diff
@@ -101,6 +102,10 @@ async def swagger_ui_redirect():
 @app.get("/", include_in_schema=False)
 @logger.catch()
 async def home(request: Request, db: Session = Depends(get_db)):
+    # delete permanently events after 30 days
+    days = 30
+    delete_events_after_optionals_num_days(days, next(get_db()))
```

Comment:
why is it on the home page?

Reply:
This function check if there is deleted events that need to be deleted permanently,
and in the home page this function will definitely run.
If you have other place to make this function run I would like to hear :)